lgc/patch/PatchEntryPointMutate.cpp

```diff
@@ -265,6 +266,13 @@ bool PatchEntryPointMutate::lowerCpsOps(Function *func) {
   if (!isCpsFunc && cpsOps.empty())
     return false;
 
+  // Get all the return instructions.
+  SmallVector<ReturnInst *> retInstrs;
+  for (BasicBlock &block : *func)
+    if (ReturnInst *ret = dyn_cast<ReturnInst>(block.getTerminator()))
+      retInstrs.push_back(ret);
```

Comment:
Instead of first collecting all `ret`s and then using the list later, just iterate over the `ret`s directly in the loop below.

Reply:
I find that iterating over the ret's later will visit the newly created tail block (have no terminator) which would need explicitly by-passing null terminator. So I just keep it as now. Please let me know if you still prefer to merge the two loops together.

Reply:
Okay, makes sense.